hotels-backend/Hotels/Controllers/HotelsController.cs

```diff
@@ -71,15 +72,27 @@ public HotelsController(HotelsDBContext context, ImageService imageService)
                         )
                     );
             }
+
             return await hotels
                 .Skip((page - 1) * returnedNumberOfItems)
                 .Take(returnedNumberOfItems)
+                .Select(hotel => new HotelDTO
+                    {
+                        Id = hotel.Id,
+                        Name = hotel.Name,
+                        Country = hotel.Country,
+                        City = hotel.City,
+                        Address = hotel.Address,
+                        Image = hotel.Image,
+                        Services = hotel.Services,
+                        RoomCount = hotel.Rooms.Count
+                    })
```

Comment:
```suggestion
                .Select(hotel => 
                    new HotelDTO
                    {
                        Id = hotel.Id,
                        Name = hotel.Name,
                        Country = hotel.Country,
                        City = hotel.City,
                        Address = hotel.Address,
                        Image = hotel.Image,
                        Services = hotel.Services,
                        RoomCount = hotel.Rooms.Count
                    }
                )
```
When creating new objects using multiline object initializer inside lambdas or method calls it is preferable to place new operator on a new line and indent {} accordingly so you can clearly see what belongs to object constructor.

For multiline method calls placing closing parenthesis on a new line can again clearly show you the boundaries of this method call.

Reply:
Understood